marketplace-ui/src/app/modules/security-monitor/security-monitor.component.scss

```diff
@@ -0,0 +1,227 @@
+body {
```

Comment:
You should not change the styles of the body tag. It can affect the marketplace website.